new-lamassu-admin/src/pages/Customers/CustomerData.js

```diff
@@ -374,6 +424,22 @@ const CustomerData = ({
     })
   }, customInfoRequests)
 
+  R.forEach(it => {
+    initialValues.smsData[it] = smsData[it]
+    smsDataElements.push({
+      name: it,
+      label: onlyFirstToUpper(it),
+      component: TextInput,
+      editable: true
+    })
+    Yup.object()
+      .shape({
+        [it]: Yup.string()
+      })
+      .required()
+      .concat(schemas.smsData)
```

Comment:
Maybe I am wrong, but this doesn't do anything? (we don't assign created object to variable)

Reply:
Ah yeah, forgot to assign it to the schemas object